index.html

```diff
@@ -215,5 +215,24 @@ <h2 class="primary-text-about">About Myself</h2>
             </ul>
         </div>
     </section>
+    <section class="contact">
+        <div class="contact-text">
+            <h2>Contact me</h2>
+            <p>
+                If you have an application you
+                are interested in developing, a
+                feature that you need built or a 
+                project that needs coding. I'd 
+                love to help with it
+            </p>
+        </div>
+        <form action="https://formspree.io/f/mrgjwpba" >
+            <input type="text" class="input" name="name" placeholder="Enter Your Name:" required maxlength="30">
+            <input type="email" class="input" name="email" placeholder="Enter Your Email:" required>
+            <textarea name="message" class="area" required maxlength="500">Write your message here!</textarea>
```

Comment:
- [OPTIONAL] It will be nice to have the text ` Write your message here!` inside a placeholder attribute like the example below. I have to delete the existing text before I could write my message.

```html
<textarea 
    name="message" 
    class="area" 
    required
    maxlength="500"
    placeholder="Write your message here!"></textarea>
```